salt/modules/cloud.py

```diff
@@ -210,6 +210,8 @@ def map_run(path=None, **kwargs):
 
         salt minionname cloud.map_run /path/to/cloud.map
         salt minionname cloud.map_run map_data='<actual map data>'
+        # to use map data in pillar under salt:cloud:maps:<map_pillar>'
```

Comment:
I believe this should be `cloud:maps` not `salt:cloud:maps`

Reply:
Heya @gtmanfred, you're absolutely right! I was referencing it's location in the salt-formula pillar by accident. I have the correction for this on my laptop already - will push ASAP.